sqlmesh/core/model/definition.py

```diff
@@ -2541,6 +2538,23 @@ def _create_model(
         for jinja_macro in jinja_macros.root_macros.values():
             used_variables.update(extract_macro_references_and_variables(jinja_macro.definition)[1])
 
+    # Merge model-specific audits with default audits
+    model_audits = kwargs.pop("audits", [])
+    default_audits = defaults.pop("audits", [])
+
+    if isinstance(model_audits, (exp.Tuple, exp.Array)):
```

Comment:
This block looks like something that could be an utility function and doesn't have to be a part of `_create_model`